django_redis/client/default.py

```diff
@@ -778,6 +797,265 @@ def make_pattern(
 
         return CacheKey(self._backend.key_func(pattern, prefix, version_str))
 
+    def sadd(
+        self,
+        key: KeyT,
+        *values: Any,
+        version: Optional[int] = None,
+        client: Optional[Redis] = None,
+    ) -> int:
+        if client is None:
+            client = self.get_client(write=True)
+
+        key = self.make_key(key, version=version)
+        encoded_values = [self.encode(value) for value in values]
+        return int(client.sadd(key, *encoded_values))
+
+    def scard(
+        self,
+        key: KeyT,
+        version: Optional[int] = None,
+        client: Optional[Redis] = None,
+    ) -> int:
+        if client is None:
+            client = self.get_client(write=False)
+
+        key = self.make_key(key, version=version)
+        return int(client.scard(key))
+
+    def sdiff(
+        self,
+        *keys: KeyT,
```

Comment:
do you know what happens in Redis if we give it multiple keys and some are duplicate?

Reply:
are you sure the type is correct for `keys`?

Reply:
today I've learned something, thanks man!